Assets/MixedRealityToolkit.Services/BoundarySystem/MixedRealityBoundarySystem.cs

```diff
@@ -570,7 +590,8 @@ public bool Contains(Vector3 location, Boundary.Type boundaryType = Boundary.Typ
             }
 
             // Handle the user teleporting (boundary moves with them).
-            location = MixedRealityToolkit.Instance.MixedRealityPlayspace.InverseTransformPoint(location);
+            // todo: update...
+            location = Playspace.InverseTransformPoint(location);
```

Comment:
>Playspace [](start = 23, length = 9)

moving to a world where less is needed, i feel like we need to start building under the assumption that someone may want to make this "Playspace" transform null. We should have appropriate null checks throughout or a default Playspace value other than null if one isn't provided.